modules/virt-about-deploying-templates-custom-namespace.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_templates/virt-about-deploying-templates-custom-namespace.adoc
+
+[id="virt-about-deploying-templates-custom-namespace_{context}"]
+= About deploying templates to a custom namespace
+
+To add templates to a custom namespace for private use, edit the `hco` custom resource (CR) and specify a custom namespace. Then, add the Red Hat templates to that custom namespace.
```

Comment:
We should drop `for private use`, and clarify SSP operator will populate the templates in the new admin-created namespace.

Reply:
Updated the content.